Readme.md

```diff
@@ -37,8 +37,8 @@ There are several fields in the `[package.metadata.bundle]` section.
 
 These settings apply to bundles for all (or most) OSes.
 
- * `name`: The name of the built application. If this is not present, then it will use the `name` value from
-           your `Cargo.toml` file.
+ * `name`: The name of the built application. If this is not present, then it will use the `name` value from `bin`
+           target in your `Cargo.toml` file.
```

Comment:
Which `bin` does it choose if there is more than one?

Reply:
First one is chosen. It is my understanding that multiple `bin` targets are not supported.